css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Media Queries - Breakpoints</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .card-container {
+      padding: 0.5rem;
+      flex-basis: 100%;
+    }
+
+    /* your code here 👇 */
+    @media screen and (min-width: 576px) {
+      .card-container {
+        flex-basis: 50%;
+      }
+
+      @media and (min-width: 768px) {
+        .card-container {
+          flex-basis: calc(100% / 3);
+        }
+      }
```

Comment:
This media query is inside the other media query, that isn't how it's meant to be 👀